src/org/opensolaris/opengrok/history/HistoryGuru.java

```diff
@@ -341,69 +343,95 @@ public void addRepositories(File[] files, Collection<RepositoryInfo> repos,
      */
```

Comment:
yep, see http://www.oracle.com/technetwork/articles/java/index-137868.html for details